server/forge/gitea/gitea.go

```diff
@@ -95,12 +88,16 @@ func (c *Gitea) URL() string {
 }
 
 func (c *Gitea) oauth2Config(ctx context.Context) (*oauth2.Config, context.Context) {
+	publicFacingOAuthURL := server.Config.Server.ForgeOAuthHost
```

Comment:
I would shorten the var name a bit and use e.g. `publicOAuthURL` but just a personal preference.

Reply:
changed it